pages/linux/tcpick.md

```diff
@@ -0,0 +1,17 @@
+# tcpick
+
+> A Linux command-line tool for packet sniffing and network traffic analysis.
+> It can capture and display TCP connections and data. It can also monitor network traffic on a specific interface, host, or port.
+> More information: <https://linux.die.net/man/8/tcpick>.
+
+- Basic usage:
+
+`tcpick -i {{interface}} -C -h {{host}} -p {{port}}`
```

Comment:
```suggestion
- Capture traffic on a specific [i]nterface, port and host::

`tcpick -i {{interface}} -C -h {{host}} -p {{port}}`
```

I think this would be better.

We prefer long options in tldr and we can use mnemonics `[]` to highlight long options whenever applicable.